.travis/test_runner.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+TESTS="$1"
+COMMAND="coverage run manage.py test --noinput --failfast --traceback --verbosity=2"
+
+/moto-s3/env/bin/moto_server s3 &
+
+if [ -z ${COMMAND_OVERRIDE} ]; then
+    $COMMAND $TESTS
+else
+    $COMMAND_OVERRIDE
+fi
```

Comment:
is there a possibility that moto_server isn't being started as daemon and thus blocking the second command from ever running?